src/test/resources/beanmapper/nested_chat_component_complex_expected.yml

```diff
@@ -24,20 +25,21 @@ message-key:
             med:
                 color: gray
                 text: med text
-                extra: 
-                - color: green
-                  text: med child
-                  extra: []
+                extra:
+                -   color: green
+                    text: med child
+                    extra: []
                 conditionals: {}
             high:
                 color: black
                 text: high text
                 extra: []
+                # Optional: additional info
```

Comment:
Add option for a comment to only appear once? Or think about it outside of this PR?